.editorconfig

```diff
@@ -0,0 +1,12 @@
+root = true
+
+[*]
+insert_final_newline = true
+
+[*.json]
+indent_style = space
+indent_size = 4
+
+[*.schema]
+indent_style = space
+indent_size = 4
```

Comment:
You didn't want to check this in did you?

Reply:
I did mean to check this in. My editor was trained to do 2 spaces instead of 4.

Reply:
The problem is that this is a "public" space.  So anyone looking at schema will also find this there which I don't think is a good idea.

Reply:
I'm not sure I follow? This configuration only affects this git repo (and only with editors that support + use editor config). It is just a formatting configuration for *.schema files that are checked in to this repository.